examples/eng_wasm_contracts/encryption/Cargo.toml

```diff
@@ -3,10 +3,8 @@ name = "contract"
 version = "0.1.0"
 
 [dependencies]
-#eng-wasm = "0.1.3"
```

Comment:
Good catch.

Reply:
Wait, bad catch :) The test of this contract will not pass, since the version with crypto service is not pushed yet. Please revert this.

Reply:
On the second thought, this worked by chance. The reason for a local dependency here is that the version of eng-wasm, which contains crypto service was not pushed yet to crates.io.  When you pushed version 0.1.4, you unintently pushed the version with crypto service, therefore this change is possible.